docs/configuration.md

```diff
@@ -56,6 +56,7 @@ spec:
     serviceCIDR: 10.96.0.0/12
     provider: kuberouter
     calico: null
+    clusterDomain: cluster.local
```

Comment:
Some explnanation would be also good to have for this. Later in this same MD file there's sub-chapters for each major part, add some explanation for this in the corresponding chapter.